cypress/integration/galleryDom.spec.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const url = 'http://localhost:8080';
```

Comment:
this should be replacable with Cypress.config().baseUrl.

Reply:
yes, it is. replaced to Cypress.config().baseUrl